Sources/Models/ALKConfiguration.swift

```diff
@@ -162,6 +162,9 @@ public struct ALKConfiguration {
         
     // If true then Navigation bar will be hidden. By default it is false
     public var hideNavigationBarOnChat : Bool = false
+    
+    // If false then new conversation button (at bottom) will not be visible. By default it is true.
+    public var hideBottomStartNewConversationButton: Bool = false
```

Comment:
We already have one hideStartButton I guess for the top one. Should we keep them separate?

Reply:
Yes we need to keep this separate. because customer wants this button, not that (top one). if we keep a single flag for both, then we can either show/hide both buttons.thatswhy created a new flag

Reply:
Okay, got it